src/actions/manager.tsx

```diff
@@ -151,6 +151,7 @@ export class ActionManager {
       const elements = this.getElementsIncludingDeleted();
       const appState = this.getAppState();
       const updateData = (formState?: any) => {
```

Comment:
When I try to edit the values for other fields, I'm getting right value, where as with a newly created input field, the formState is not available